src/components/Input.tsx

```diff
@@ -5,30 +5,51 @@ const Input = ({
   placeholder,
   helpText,
   disabled,
+  className,
+  value,
+  onChange,
+  mandatoryField,
+  errorMessage,
+  optional = false,
 }: {
   type: string;
   name: string;
   label: string;
   placeholder?: string;
   helpText?: string;
   disabled?: boolean;
+  className?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  mandatoryField?: boolean;
```

Comment:
let's rename `mandatoryField` to just `required`
```suggestion
  required?: boolean;
```